File: 05 SWEA/D1/Java/src/Prob2068.java

/**
* title  : 2068. 최대수 구하기 (D1)
* time   : 103ms
* memory : 18612MB
*/

import java.io.*;
import java.util.*;
 
public class Prob2068 {
    
    public static void main(String[] args) throws IOException {
        
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int T = Integer.parseInt(br.readLine());
        for (int t = 1; t <= T; t++) {

            int answer = 0;

            StringTokenizer st = new StringTokenizer(br.readLine());
            for (int i = 0; i < 10; i++) {
                answer = Math.max(answer, Integer.parseInt(st.nextToken()));
            }

            sb.append("#" + t).append(" " + answer).append("\n");
        }

        System.out.println(sb);
    }
}

File: 05 SWEA/D1/Python/Prob2068.py

"""
title  : 2068. 최대수 구하기 (D1)
time   : 122ms
memory : 44796kb
"""

def solution():

    T = int(input())
    for tc in range(1, T+1):

        answer = max(list(map(int, input().split())))
        print(f'#{tc} {answer}')

solution()